docs/getting_started/authentication.md

```diff
@@ -0,0 +1,40 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Authentication
+
+The Authentication mechanism is based on a bearer token.
+How does it work?
+Both Jellyfish and Business Logic are configured with the same secret.
+This secret is used for authorizing HTTP requests between them (via HTTP authorization request header).
+When a new client wants to join a room, Business Logic (assuming the client is authorized to join) requests adding a new peer and receives a token generated by Jellyfish.
+The token must then be passed to the client, that will use it to connect to the room as a new peer. 
+The token encodes both peer and room IDs.
+The token exchange is handled by our Client SDKs when connecting.
+Under the hood, Jellyfish expects the token in the first message after opening the Web Socket connection.
+
+Token encodes peer id and room id. 
+
+1. Run Jellyfish
+
+```bash
+JELLYFISH_SECRET=mySecret mix phx.server
```

Comment:
Such env var is not read, but there is a `TOKEN`. Is this going to be changed? If so, I'd recommend using `SERVER_API_SECRET`

Reply:
Let's change this in the source code at first